### ISSUE TITLE
Extraneous prints in ChemEnv and TR for OVP code

### ISSUE THREAD

**Issue by mathomp4** [GEOS-ESM/GEOSchem_GridComp#117]
In recent runs of GEOSgcm there is now a new print appearing due to ChemEnv from v1.4.2:
```
 in CHEMENV the RUN_DT and CHEM_DT values are:          450         450
```
This seems due to this line in `GEOS_ChemEnvGridComp.F90`:
https://github.com/GEOS-ESM/GEOSchem_GridComp/blob/695c8d1bd0453d3dd7bfb5a74a80b3326ee21300/GEOS_ChemEnvGridComp.F90#L517

Good thing is this is done only once during a run, but it is printed out by ***every*** process. Obviously we could just comment, remove, or `if (MAPL_AM_I_ROOT())` this print. But I guess it didn't ping my radar at PR time because it's in a code block that is only OVP related. The thing is...I'm not deliberately asking to run any OVP code. 

I guess the questions for @mmanyin (or whoever brought this code in) is: Print aside, is this code expensive? That is, it seems that *everyone* will now call the OVP code, everywhere. I suppose I haven't noticed the cost, so I guess it isn't?

Also, should we remove/comment out all the other currently active OVP prints I see:
https://github.com/GEOS-ESM/GEOSchem_GridComp/blob/695c8d1bd0453d3dd7bfb5a74a80b3326ee21300/GEOS_ChemEnvGridComp.F90#L523
https://github.com/GEOS-ESM/GEOSchem_GridComp/blob/695c8d1bd0453d3dd7bfb5a74a80b3326ee21300/TR_GridComp/TR_GridCompMod.F90#L1541
https://github.com/GEOS-ESM/GEOSchem_GridComp/blob/695c8d1bd0453d3dd7bfb5a74a80b3326ee21300/TR_GridComp/TR_GridCompMod.F90#L1551

(Note the ones in TR have been there since Git began, so I guess I don't notice them!)

@mmanyin Would you like me to make a PR to quiet these prints?

**Comment by mmanyin** [comment]
@mathomp4 The OVP diagnostics are only computed if the HISTORY includes them; no significant overhead.  I will submit a PR that silences the PRINT statements, and eliminates some unneeded comments.

**Comment by mathomp4** [comment]
@mmanyin Thanks! I just wanted to make sure. I was so confused when I saw the print was in a OVP section and I was thinking "Have I been turning this on by mistake?" (Very possible. My makeoneday script has had some fun bugs exposed recently! 😄 )

**Comment by mmanyin** [comment]
This issue is addressed by #118

**Comment by mathomp4** [comment]
@mmanyin Fixed this a long time ago. I guess GitHub never closed this. Closing!
